source/_integrations/homewizard.markdown

```diff
@@ -71,3 +71,15 @@ The Wifi Energy Socket (`HWE-SKT`) outlet state can be controlled the switch pla
 
 - **Switch**: Controls the outlet state of the Energy Socket. This switch is locked out when `Switch Lock` is turned on. 
 - **Switch lock**: Forces the outlet state in the `on` position and disables the physical button. This option is useful when the socket is used for a device that must not be turned off, such as a refrigerator.
+
+## Cloud communication
+
+The HomeWizard Energy devices are designed to work with the HomeWizard Energy app and required communication with the HomeWizard cloud to make them function with the app. The `Enable cloud` configuration toggle can be used to turn off all communication with the HomeWizard cloud, making the device fully local. The device cannot communicate with the app and the device won't receive any future firmware updates.
+
+Cloud communication is restored after factory reset or when the device is put in pairing mode.
```

Comment:
The parent PR adds a switch (meaning it can be switched on/off). Therefore I do not fully understand this line. Once turned off, you can turn it on again without factory resetting?

Reply:
Hmm. Maybe it is my English..

I think it is important for the user to know how to re-enable the cloud without having access to the toggle (eg. when selling/giving away the device, or when the user has no access to Home Assistant somehow).

Any ideas how we can communicate that?

Reply:
Awesome!